pkg/crc/preflight/preflight_checks_tray_windows.go

```diff
@@ -61,11 +60,13 @@ func fixTrayInstalled() error {
 
 	// sanitize password
 	password = escapeWindowsPassword(password)
+	username := goos.Getenv("USERNAME")
```

Comment:
Maybe not relevant here I don't know but you can probably use `user.Current()`

Reply:
Yes, but `user.Username` returns full display name with the machine name like `DESKTOP-HFRVFP1\Anjan` i only require the account part like from `USERNAME` variable.. 
```posh
PS C:\Users\Anjan\github.com\code-ready\crc> $env:USERNAME
Anjan
```